src/output.c

```diff
@@ -95,11 +95,12 @@ cmd_outputs(gcc_unused int argc, gcc_unused char **argv,
 		/* We increment by 1 to make it natural to the user  */
 		int id = mpd_output_get_id(output) + 1;
 		const char *name = mpd_output_get_name(output);
+        const char *plugin = mpd_output_get_plugin(output);
 
 		if (mpd_output_get_enabled(output)) {
-			printf("Output %i (%s) is enabled\n", id, name);
+			printf("Output %i (%s: %s) is enabled\n", id, plugin, name);
```

Comment:
This is unrelated to partitions, and doesn't belong in this commit.

Reply:
Fixed.